### ISSUE TITLE
[Bug]: pgSQL - Chunks fail to save with "extended protocol limited to 65535 parameters"

### ISSUE THREAD

**Issue by marshyonline** [data-preservation-programs/singularity#171]
### Description

```text
2023-08-07T12:30:34.904+1000    ERROR   database        database/util.go:69     UPDATE "item_parts" SET "chunk_id"=282 WHERE id IN (288630,288631,288632,288633,288634,288635,288636,288637,288638,288639,288640,288641,288642,288643,288644,288645,288646,288647,288648,288649,288650,288651,288652,288653,288654,288655,288656,288657,288658,288659,288660,288661,288662,288663,288664,288665,288666,288667,288668,288669,288670,288671,288672,288673,288674,288675,288676,288677,288678,288679,288680,288681,288682,288683,288684,288685,288686,288687,288688,288689,288690,288691,288692,288693,288694,288695,288696,288697,288698,288699,288700,288701,288702,288703,288704,288705,288706,288707,288708,288709,288710,288711,288712,288713,288714,288715,288716,288717,288718,288719,288720,288721,288722,288723,288724,288725,288726,288727,288728,288729,288730,288731,288732,288733,288734,288735,288736,288737,288738,288739,288740,288741,288742,288743,288744,288745,288746,288747,288748,288749,288750,288751,288752,288753,288754,288755,288756,288757,288758,288759,288760,288761,288762,288763,288764,288...(trimmed)  {"rowsAffected": 0, "elapsed": 0.716359926, "err": "extended protocol limited to 65535 parameters"}
2023-08-07T12:30:34.906+1000    ERROR   datasetworker   datasetworker/datasetworker.go:201      failed to scan  {"workerID": "0391e175-97a6-4e20-a7e8-5acaf0111cf6", "error": "failed to save chunking: failed to create chunk: failed to update items: extended protocol limited to 65535 parameters", "errorVerbose": "extended protocol limited to 65535 parameters\nfailed to update items\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).chunkOnce.func2.1\n\t/root/SinuglarityV2/service/datasetworker/scan.go:156\ngorm.io/gorm.(*DB).Transaction\n\t/root/go/pkg/mod/gorm.io/gorm@v1.25.2-0.20230530020048-26663ab9bf55/finisher_api.go:647\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).chunkOnce.func2\n\t/root/SinuglarityV2/service/datasetworker/scan.go:141\ngithub.com/avast/retry-go.Do\n\t/root/go/pkg/mod/github.com/avast/retry-go@v3.0.0+incompatible/retry.go:127\ngithub.com/data-preservation-programs/singularity/database.DoRetry\n\t/root/SinuglarityV2/database/util.go:23\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).chunkOnce\n\t/root/SinuglarityV2/service/datasetworker/scan.go:140\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).scan\n\t/root/SinuglarityV2/service/datasetworker/scan.go:70\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).run\n\t/root/SinuglarityV2/service/datasetworker/datasetworker.go:199\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1594\nfailed to create chunk\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).chunkOnce\n\t/root/SinuglarityV2/service/datasetworker/scan.go:163\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).scan\n\t/root/SinuglarityV2/service/datasetworker/scan.go:70\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).run\n\t/root/SinuglarityV2/service/datasetworker/datasetworker.go:199\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1594\nfailed to save chunking\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).scan\n\t/root/SinuglarityV2/service/datasetworker/scan.go:72\ngithub.com/data-preservation-programs/singularity/service/datasetworker.(*DatasetWorkerThread).run\n\t/root/SinuglarityV2/service/datasetworker/datasetworker.go:199\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1594"}
```

### Steps to Reproduce

1. Run below commands
```
singularity ...
```
2. Go to '....'
3. See error

### Version

singularity v0.2.47-e01eb47

### Operating System

Linux

### Database Backend

PostgreSQL

**Comment by marshyonline** [comment]
We were testing #137 with a large dataset (25TiB) and found this error while scanning the DS.
It _looks_ like the may have more than 65535 files in a single chunk and thus this chunk failed to store to the DB and stopping the scanning.

**Comment by xinaxu** [comment]
This is fixed in #182
